Annotations/AddAnnotation/MainWindow.xaml.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.Drawing;
 using System.IO;
+using System.Runtime.InteropServices;
```

Comment:
Why this namespace is added

Reply:
Removed newly added namespace is completed